sporkfish/searcher/move_ordering.py

```diff
@@ -69,3 +73,50 @@ def evaluate(self, board: Board, move: chess.Move) -> float:
             ]
         else:
             return 0
+
+
+class KillerMoveHeuristic(MoveOrder):
+    def __init__(self, max_depth: int) -> None:
+        # Store up to two killer moves for each depth
+        # Using a fixed size per depth for simplicity; could be dynamic based on actual usage
+        self._killer_moves: List[List[Optional[chess.Move]]] = [
+            [None, None] for _ in range(max_depth + 1)
```

Comment:
NB: fixed max_depth + 1, because it should include max_depth...

Reply:
makes sense - nice catch